Source/DafnyLanguageServer/Workspace/TextDocumentLoader.cs

```diff
@@ -90,9 +90,9 @@ ILogger<ITextDocumentLoader> logger
           ? null
           : symbolTableFactory.CreateFrom(program, compilationAfterParsing, cancellationToken);
         if (errorReporter.HasErrors) {
-          statusPublisher.SendStatusNotification(compilation, CompilationStatus.ResolutionFailed);
+          var _4 = statusPublisher.SendStatusNotification(compilation, CompilationStatus.ResolutionFailed);
         } else {
-          statusPublisher.SendStatusNotification(compilation, CompilationStatus.CompilationSucceeded);
+          var _5 = statusPublisher.SendStatusNotification(compilation, CompilationStatus.CompilationSucceeded);
```

Comment:
nit: These numbered ignored-variables are a bit distracting. It looks like you can just write `_ = ReturnsUnneededValue();` (no `var`) in C# 7.0 and above: https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/functional/discards

Reply:
Awesome, thanks !